cpp/ps0005/ps0005_soin_cpp.cpp

```diff
@@ -1,31 +1,43 @@
 #include <iostream>
-#include<conio.h>
+#include <conio.h>
 using namespace std;
 int main(){
-	int number1, number2;
-	char x;
+	// prompting user for 2 operands and an operator 1.5/2
+	double number1, number2;
+	char op;
 	cout << "enter the two operands\n";
 	cin >> number1;
 	cin >> number2;
 	cout << "enter the operator + - % * / \n";
-	cin >> x;
-
-	if (x == '+')
+	cin >> op;
```

Comment:
- [x] declared variables to store the user input
- [ ] chose suitable data types for the variables
- [x] prompted user to enter data
- [x] used `std::cin` in a right way to get user's input